lute/backup/routes.py

```diff
@@ -7,12 +7,24 @@
 import traceback
 from flask import Blueprint, current_app, render_template, request, jsonify, redirect
 from lute.models.setting import BackupSettings
-from lute.backup.service import create_backup, skip_this_backup
+from lute.backup.service import create_backup, skip_this_backup, list_backups
 
 
 bp = Blueprint("backup", __name__, url_prefix="/backup")
 
 
+@bp.route("/index")
+def index():
+    """
+    List all backups.
+    """
+    settings = BackupSettings.get_backup_settings()
+    backups = list_backups(settings.backup_dir)
+    backups.sort(reverse=True)
```

Comment:
For longtime users like me, the list might get long.  The automatic backup list is pruned regularly, but the manual ones are not.  Limiting the list should be for a later iteration.

Reply:
Yep, I was thinking maybe there would be a delete option given for manual backups in the future.